src/lib.rs

```diff
@@ -213,14 +209,11 @@ impl DocData {
         for def in defs.iter() {
```

Comment:
The reason being is defs isn't used after this loop at all so consuming the values should be fine.